pkgs/by-name/wu/wush/package.nix

```
@@ -0,0 +1,36 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+let
+  version = "0.1.2";
+in
+buildGoModule {
+  pname = "wush";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "coder";
+    repo = "wush";
+    rev = "v${version}";
+    hash = "sha256-r6LKEL9GxyiyQgM4AuLU/FcmYKOCg7EZDmAZQznCx8E=";
+  };
+
+  vendorHash = "sha256-e1XcoiJ55UoSNFUto6QM8HrQkkrBf8sv4L9J+7Lnu2I=";
+
+  ldflags = [
+    "-s -w -X main.version=${version}"
+  ];
+
+  CGO_ENABLED = 0;
+
+  meta = with lib; {
+    homepage = "https://github.com/coder/wush";
+    description = "CLI P2P file transfer tool over wireguard";
```

```suggestion
    description = "Transfer files between computers via WireGuard";
```
To stay in line with the homebrew pkg https://github.com/Homebrew/homebrew-core/pull/183296/files#diff-abeabfe2c810a79c21facd1fe5fc7f0d6f805b11318540027fc2e55866f8e44aR2